Server_Side_TLS.mediawiki

```diff
@@ -35,170 +35,167 @@ The ordering of a ciphersuite is very important because it decides which algorit
 == <span style="color:green;">'''Modern'''</span> compatibility ==
 For services that don't need backward compatibility, the parameters below provide a higher level of security. This configuration is compatible with Firefox 27, Chrome 30, IE 11 on Windows 7, Edge, Opera 17, Safari 9, Android 5.0, and Java 8.
 
-* Ciphersuites: '''ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256'''
-* Versions: '''TLSv1.2'''
-* TLS curves: '''prime256v1, secp384r1, secp521r1'''
-* Certificate type: '''ECDSA'''
-* Certificate curve: '''prime256v1, secp384r1, secp521r1'''
+* Ciphersuites: '''ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256'''
```

Comment:
Sorry I made a mistake when doing OpenSSL speed test.
In OpenSSL 1.1.0 I did: 
`/path/to/openssl speed -evp chacha20`
instead of
`/path/to/openssl speed -evp chacha20-poly1305`
So my results for chacha20 was faster than aes-256-gcm but slower than aes-128-gcm.
chacha20-poly1305 is slower than both.
Speed test result:
```
type             16 bytes     64 bytes    256 bytes   1024 bytes   8192 bytes  16384 bytes
aes-128-gcm     215564.66k   516407.32k  1090354.18k  1310255.45k  1575836.68k  1826177.02k
aes-256-gcm     172686.72k   471236.02k   948903.34k  1151707.48k  1350737.92k  1275957.43k
chacha20        169704.14k   289461.82k   669630.89k  1384385.54k  1431011.33k  1535246.34k
chacha20-poly1305   131241.25k   237345.05k   473652.14k   882510.85k   881535.66k   924150.44k
```